scenariolib/event_pageview.go

```diff
@@ -69,8 +69,13 @@ func (view *ViewEvent) send(v *Visit) error {
 	v.DecorateEvent(event.ActionEvent)
 	v.DecorateCustomMetadata(event.ActionEvent, view.CustomData)
 
+	if view.PageViewField == "" {
```

Comment:
Put this in the IsValid() function above.
I think a PageView event is simply not valid without a PageViewField right?

Reply:
I moved it.